docs/source/notes/autograd.rst

```diff
@@ -558,35 +558,35 @@ Now using Wirtinger derivative definition, we can write:
 
     .. math::
         \begin{aligned}
-            \frac{\partial L}{\partial s} = 1/2 * (\frac{\partial L}{\partial u} - \frac{\partial L}{\partial v} j) \\
-            \frac{\partial L}{\partial s^*} = 1/2 * (\frac{\partial L}{\partial u} + \frac{\partial L}{\partial v} j)
+            \frac{\partial L}{\partial s} = 1/2 * \left(\frac{\partial L}{\partial u} - \frac{\partial L}{\partial v} j\right) \\
+            \frac{\partial L}{\partial s^*} = 1/2 * \left(\frac{\partial L}{\partial u} + \frac{\partial L}{\partial v} j\right)
         \end{aligned}
 
 It should be noted here that since :math:`u` and :math:`v` are real
 functions, and :math:`L` is real by our assumption that :math:`f` is a
 part of a real valued function, we have:
 
     .. math::
-        (\frac{\partial L}{\partial s})^* = \frac{\partial L}{\partial s^*}
+        \left( \frac{\partial L}{\partial s} \right)^* = \frac{\partial L}{\partial s^*}
         :label: [2]
 
-i.e., :math:`\frac{\partial L}{\partial s}` equals to :math:`grad\_output^*`.
+i.e., :math:`\frac{\partial L}{\partial s}` can be expressed by :math:`\frac{\partial L}{\partial s^*}` which is the incoming :math:`grad\_output`.
```

Comment:
uhh I prefer the original version because we already see this from the equation above